docs/examples/video/video_label_example.py

```diff
@@ -0,0 +1,80 @@
+#!/bin/env python
```

Comment:
A couple of things:
- I think docs/examples/video/video_example.py can be removed in favor of `video_label_example.py`
- in the bellow example you should return labels as well and do something with them
- I think it should be reworked to jupyter notebook and included in docs like `video_reader_simple_example.ipynb` is (in docs/examples/index.rst)

Reply:
>in the bellow example you should return labels as well and do something with them

In `video_label_example.py` we are returning the label and printing it. 
We could additionally save the extracted frames to different directories based on labels.
https://github.com/NVIDIA/DALI/blob/6f61997f4da71e0eaddf3c6871c9bf0ba6535c56/docs/examples/video/video_label_example.py#L58-L65

Reply:
I missed that because you returning the whole `output` from `self.input` call, it is fine. We usually recommend to return each piece separately for the readability:
```
frames, labels = self.input(name="Reader")
```
I think there is no need to save them. Still I would prefer to see a jupyter notebook - like this https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/docs/examples/video/video_reader_simple_example.html (it is more likely that user will check this than spot this python example somewhere in the GitHub).

Reply:
Added a jupyter notebook
https://github.com/NVIDIA/DALI/blob/94718ac73d1009a52084f9653e5376f03dc9ced1/docs/examples/video/video_reader_label_example.ipynb